src/fidesops/graph/config.py

```diff
@@ -219,6 +220,14 @@ def cast(self, value: Any) -> Optional[Any]:
         return value
 
 
+@dataclass
+class MaskingOverride:
```

Comment:
As far as I can tell, this is only being used in iteration in the query config as a temporary storage of (datatype, length). Maybe this class is not needed?

Reply:
With a class, we have the benefit of type hints (vs using a `namedtuple`), see https://stackoverflow.com/a/50038614/4957420